engine/launcher/src/main/scala/org/enso/launcher/Launcher.scala

```diff
@@ -207,7 +208,7 @@ case class Launcher(cliOptions: GlobalCLIOptions) {
   def runRepl(
     projectPath: Option[Path],
     versionOverride: Option[SemVer],
-    logLevel: LogLevel,
+    logLevel: Level,
```

Comment:
I think having our own logging abstractions has benefits, i.e. whenever we want to change the implementation, the interface will stay the same. Do we really want to swap it with slf4j?